status-group-chats-spec.md

```diff
@@ -28,177 +28,105 @@ This documents describes the group chat protocol used by the status application.
 
 ## Membership updates
 
-Membership updates messages are used to propagate group chat membership changes. The transit format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
-
-The format is:
+Membership updates messages are used to propagate group chat membership changes. The protobuf format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
+
+The protobuf messages are:
+
+```protobuf
```

Comment:
1. Could we specify protobuf version used in these fragments? Either globally or per section

2. If we have a package name, can we specify these? https://developers.google.com/protocol-buffers/docs/proto If we don't, could we consider adding it?

Reply:
I will add the version, currently we use `protobuf` as a package name (in go it does not have change the generated code, so we can pick anything), what should we go for? (I might update the specs though in a separate PR, once the change is made)

Reply:
Yeah we should defo specify, in MVDS we also specify which version we use.

Reply:
@cammellos @oskarth you can add `syntax = "proto3";` to the protocol buffer itself as the first line before the message.

Reply:
just to clarify, we do always specify both `syntax` (`proto3`) and package (`protobuf`, the package name we use is just `protobuf`), just wondering whether we should use a more meaningful package name

Reply:
What about `status.messaging`? Looking at https://github.com/status-im/status-go/tree/baa0767c263fa25fdd8d90a42d13f1f1fffce804/protocol/protobuf it seems like there are 3 short-ish protobufs, so having one namespace for all of them seems good enough for them. Once we do different domains, we can have `status.foo`, and we'll also have `vac.waku` etc.